gdp/small_batch/gdp_small_batch.py

```diff
@@ -273,27 +326,24 @@ def external_ref(m, x, logic_expr=None):
     p = 0
     for j in m.j:
         ext_var[j] = x[p]
-        p = p+1
+        p = p + 1
 
     for k in m.k:
         for j in m.j:
             if k == ext_var[j]:
                 m.Y[k, j].fix(True)
-                m.Y_exists[k, j].indicator_var.fix(
-                    True)  # IS THIS REQUIRED????
-                m.Y_not_exists[k, j].indicator_var.fix(
-                    False)  # IS THIS REQUIRED????
+                m.Y_exists[k, j].indicator_var.fix(True)  # IS THIS REQUIRED????
```

Comment:
Is this comment still relevant? Try changing it to a note at least

Reply:
I managed to add the documentation.